pages/platform/data-processing/36_HOWTO_submit-job-cli/guide.en-gb.md

```diff
@@ -0,0 +1,127 @@
+```
+[ovh]
+; configuration specific to 'ovh-eu' endpoint as it's the only one available for now
+endpoint=ovh-eu
+application_key=my_app_key
+application_secret=my_application_secret
+consumer_key=my_consumer_key
+```
+
+### Step 3: Upload your application code to Object Storage
+Before running your job in Data Processing platform, you will need to create a container in OVHcloud Object Storage. 
+You can work with your Object Storage using either the OVHcloud Manager or the Openstack Horizon dashboard.
+
+Please see [Creating Storage Containers in Customer Panel](../../storage/pcs/create-container/){.external} or [Create an object container in Horizon](../../storage/create_an_object_container/){.external} for more details.
+
+You can also manage your Object storage through command line with the [Openstack Swift API](https://docs.ovh.com/gb/en/public-cloud/getting_started_with_the_swift_api/){.external} 
+
+When it is created, upload your application code in your container. If you don't have any application code, you can still try the CLI with the examples files provided inside the ``testdata`` directory of the [GitHub project](https://github.com/ovh/data-processing-spark-submit){.external}.
+If you want to submit a python job, do not forget to upload your environment.yml file (see [How to generate environment file for Python jobs](../generate-environment){.external})
+
+>[!primary]
+>
+> If you only have your application code to upload, you can use the auto-upload feature of the CLI instead of uploading it manually (see **step 4: Submit a job - Optionally use auto-upload**).
+
+
+### Step 4: Submit a job
+
+Now everything is ready, let's submit a job !
+
+To launch a job with the Data Processing CLI, you have to run the executable file you previously built with your job configurations as parameters.
+
+Here is an example of command you could run to submit a SparkPi job in java/scala:
+
+```shell-session
+$ ./ovh-spark-submit --projectid yourProjectId --class org.apache.spark.examples.SparkPi --driver-cores 1 --driver-memory 4G --executor-cores 1 --executor-memory 4G --num-executors 1 swift://odp/spark-examples.jar 1000
+```
+
+In this example, the application code ``spark-examples.jar``is stored in the ``odp`` container on Object Storage, the Spark driver and its executor have 1 core and 4 gibibytes of memory. Once the Spark cluster deployed the script will run with ``1000`` as argument.
+
+>[!primary]
+>
+> Some of the parameters can be set as environment variables, such as ** --projectid**.
+
+The ovh-spark-submit CLI provides a part of the parameters of spark-submit to configure your job.
+If you want to know more about these parameters run:
+
+```shell-session
+$ ./ovh-spark-submit -h
+```
+
+While your job is running, you can watch logs in your terminal or access Spark UI through this URL: 
+
+``https://adc.{region}.dataconvergence.ovh.com/{your-job-id}/jobs/``
```

Comment:
Where can I find my {region} values ?
Also add a link to the page with limitations?